TypewiseAlert.Test/TypewiseAlertTest.cs

```diff
@@ -8,8 +9,34 @@ public class TypewiseAlertTest
     [Fact]
     public void InfersBreachAsPerLimits()
     {
-      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+      Assert.True(TypewiseAlert.inferBreach(12, 20, 30) is LowBreachType);
+      Assert.True(TypewiseAlert.inferBreach(35, 20, 30) is HighBreachType);
+      Assert.True(TypewiseAlert.inferBreach(25, 20, 30) is NormalBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new PassiveCooling(), 50) is HighBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new PassiveCooling(), 25) is NormalBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new HighActiveCooling(), 50) is HighBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new HighActiveCooling(), -1) is LowBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new MediumActiveCooling(), 50) is HighBreachType);
+      Assert.True(TypewiseAlert.classifyTemperatureBreach(new MediumActiveCooling(), 30) is NormalBreachType);
+
+
+      var alertTarget = new Mock<AlertTarget>();
+      var batteryChar1 = new TypewiseAlert.BatteryCharacter() { coolingType = new PassiveCooling() };
+
+      TypewiseAlert.checkAndAlert(alertTarget.Object, batteryChar1, 25);
+      alertTarget.Verify(func => func.SendTarget(It.IsAny<NormalBreachType>()),Times.Exactly(1));
```

Comment:
how is it verified that the Alert target has been delivered the intended message? is there a test case that checks whether email target has received Low, high or normal breach type message?